vizro-core/tests/integration/test_examples.py

```diff
@@ -33,6 +33,8 @@ def dashboard(request, monkeypatch):
     return app["dashboard"]
 
 
+# Ignore deprecation warning until this is solved: https://github.com/plotly/dash/issues/2590
+@pytest.mark.filterwarnings("ignore:HTTPResponse.getheader()")
```

Comment:
Currently only ignoring it locally for this test as this is the only place where we use dash_duo. If at some point we use dash_duo in other places as well, we can decide to move this ignore one level up etc.